packages/perseus/src/widgets/cs-program/cs-program.tsx

```diff
@@ -121,15 +138,15 @@ class CSProgram extends React.Component<any> {
         return Changeable.change.apply(this, args);
     };
 
-    simpleValidate: (arg1: any) => any = (rubric) => {
+    simpleValidate(rubric): PerseusScore {
```

Comment:
I _wish_ we could figure out how to enforce that our Perseus widget React component conform to an interface. 🤔

Reply:
Yeah or if we didn't have external methods on components 😅